ext/gmp/tests/gmp_cryptography.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+test some of the simple operations done in ECC cryptography (GH-16870)
+--EXTENSIONS--
+gmp
+--FILE--
+<?php
+$big_128 = gmp_init("275458973678998164972199298356823445708", 10);
+$big_256 = gmp_init("105897378200704506708747774951422238999742054168150946506894304230846229930690", 10);
+$big_384 = gmp_init("6416793922163250205142333121845147158821454739198730072980369042145699078647151545580979451456624745026383368515606", 10);
+$big_521 = gmp_init("1780138296718969229154737597263093326930734138014367844231086609903991655888322554437298817404082596582997382075361284417827317144352840347188199593652743789", 10);
+// in ecc crypto key lengths of 256, 384 and 521 (yes, 521, not 512) are typical,
+// and the calculations seem to include squaring and cubing the key,
+// so test those operations
+var_dump((string) gmp_pow($big_128, 2));
+var_dump((string) gmp_pow($big_128, 3));
+#var_dump((string) gmp_pow($big_128, 65537));
+
+var_dump((string) gmp_pow($big_256, 2));
+var_dump((string) gmp_pow($big_256, 3));
+#var_dump((string) gmp_pow($big_256, 65537));
+
+var_dump((string) gmp_pow($big_384, 2));
+var_dump((string) gmp_pow($big_384, 3));
+#var_dump((string) gmp_pow($big_384, 65537));
+
+var_dump((string) gmp_pow($big_521, 2));
+var_dump((string) gmp_pow($big_521, 3));
+#var_dump((string) gmp_pow($big_521, 65537));
```

Comment:
Normal that the 3rd call in each block is commented out?

Reply:
That was some experiment by me, that was not meant to be committed. Thought I had that stashed, but seems like I hadn't.
I've removed them.